src/app/components/community/channel/card/card.ts

```diff
@@ -0,0 +1,37 @@
+import Vue from 'vue';
+import { Component, Prop } from 'vue-property-decorator';
+import { MediaItem } from '../../../../../_common/media-item/media-item-model';
+import { AppTooltip } from '../../../../../_common/tooltip/tooltip';
+
+@Component({
+	directives: {
+		AppTooltip,
+	},
+})
+export default class AppCommunityChannelCard extends Vue {
+	@Prop(MediaItem)
+	backgroundItem?: MediaItem;
+
+	@Prop(String)
+	title!: string;
```

Comment:
Just making sure, the reason you pass in these values as props instead of an actual channel model because this is meant to also display the featured channel, which isn't a real channel resource, yeh?

Reply:
Yes.